cumulusci/tasks/metadata_etl/base.py

```diff
@@ -305,3 +306,48 @@ def _transform(self):
                 removed_api_names.add(api_name)
 
         self.api_names = self.api_names - removed_api_names
+
+
+class UpdateMetadataFirstChildTextTask(MetadataSingleEntityTransformTask):
+    """
+    Updates the metadata's first child tag's text.
+    Does not support tags with complex/object-like values.
+    """
+
+    task_options = {
+        "entity": {"description": "Metadata Entity", "required": True},
+        "tag": {
+            "description": "Targeted tag. The text of the first instance of this tag within the metadata entity will be updated.",
+            "required": True,
+        },
+        "value": {
+            "description": "Desired value to set for the targeted tag's text. This value is namespace-injected.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        super()._init_options(kwargs)
+        self.entity = self.options.get("entity")
+        self.options["value"] = self._inject_namespace(self.options.get("value"))
+
+    def _transform_entity(
+        self, metadata: MetadataElement, api_name: str
+    ) -> MetadataElement:
+        """
+        Finds metadata's first child with tag.  If no child is found, appends
+        a new child with tag.  Then updates child's text as the value option.
+        """
+        tag = self.options.get("tag")
+
+        attribute = metadata.find(tag)
+        if not attribute:
```

Comment:
```suggestion
        if attribute is None:
```

(I don't think it actually matters in this case, but it's a good habit to get into because in Python there can be non-None objects which are considered false when evaluated as a boolean)

Reply:
Done.  Will push momentarily.